### ISSUE TITLE
Line break in table in module/package documentation not rendered correctly

### ISSUE THREAD

**Issue by asfalcone** [Kotlin/dokka#1896]
**Describe the bug**
`<br>` appears as text instead of an HTML tag when included inside of a table cell in markdown file used as module or package documentation

**Expected behaviour**
A line break appears in the table cell

**Screenshots**
![image](https://user-images.githubusercontent.com/1848683/116327542-fdccfc00-a794-11eb-959d-a268b991c99b.png)
![image](https://user-images.githubusercontent.com/1848683/116327597-2228d880-a795-11eb-83a3-a11f4abd7e6e.png)

**_Rendered correctly in Github:_**
|  | APIs |
| ----- | ----- |
| Filled | [Icons.Default]<br>[Icons.Filled] |
| Outlined | [Icons.Outlined] |
| Rounded | [Icons.Rounded] |
| Two tone | [Icons.TwoTone] |
| Sharp | [Icons.Sharp] |

**To Reproduce**
pass a .md file via -includes to Dokka: 
```
|  | APIs |
| ----- | ----- |
| Filled | [Icons.Default]<br>[Icons.Filled] |
| Outlined | [Icons.Outlined] |
| Rounded | [Icons.Rounded] |
| Two tone | [Icons.TwoTone] |
| Sharp | [Icons.Sharp] |
```

**Installation**
- Operating system: macOS/Windows/Linux
- Build tool: Gradle v6.8.0/Maven/Cli/3rd party plugin
- Dokka version: 1.4.30

**Comment by MarcinAman** [comment]
@asfalcone could you try it out with 1.4.32? I think this should not be a problem after this: https://github.com/Kotlin/dokka/pull/1805

**Comment by asfalcone** [comment]
I am not seeing this fixed with 1.4.32

**Comment by kamildoleglo** [comment]
Fix confirmed in `1.4.32`. Basically `ContentText` has now an extra parameter `HtmlContent` if it should be treated as an HTML entity, based on DocTag's `param` `content-type == "html"`
